src/main.rs

```diff
@@ -162,7 +151,10 @@ fn main() {
             exit(-3);
         });
 
-    let build_path = format!("~/remote-builds/{:?}/", project_name);
+    // generate a unique build path by using the hashed project dir as folder on the remote machine
+    let mut hasher = DefaultHasher::new();
+    project_dir.hash(&mut hasher);
+    let build_path = format!("~/remote-builds/{:?}/", hasher.finish());
```

Comment:
I guess there wasn't a good reason to format using `Debug` in the first place, but can we get rid of it in this PR please? Changing `{:?}` to `{}` doesn't affect the outcome and seems much cleaner.

Reply:
Ahh that was just by mistake...

Reply:
Yeah, not your fault, it was actually in the original code (for no goodreason afaik) :D I just wanted to get rid of it if we touch the code anyway.